redirects/linux.nginx

```diff
@@ -0,0 +1,344 @@
+# =====================================================
+
+location /minio/baremetal/object-retention/minio-object-retention.html {
+     return 301 https://min.io/minio/docs/linux/administration/object-management.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/object-retention/bucket-versioning.html {
+     return 301 https://min.io/minio/docs/linux/administration/object-management/object-versioning.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/object-retention/minio-object-locking.html {
+     return 301 https://min.io/minio/docs/linux/administration/object-management/object-retention.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/lifecycle-management/lifecycle-management-overview.html {
+     return 301 https://min.io/minio/docs/linux/administration/object-management/object-lifecycle-management.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/lifecycle-management/transition-objects-to-s3.html {
+     return 301 https://min.io/minio/docs/linux/administration/object-management/transition-objects-to-s3.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/lifecycle-management/transition-objects-to-gcs.html {
+     return 301 https://min.io/minio/docs/linux/administration/object-management/transition-objects-to-gcs.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/lifecycle-management/transition-objects-to-azure.html {
+     return 301 https://min.io/minio/docs/linux/administration/object-management/transition-objects-to-azure.html?ref=docs-redirect;
+}
+
+# Replication Redirects from NuDocs 1 to NuDocs 2
+# ===============================================
+
+location /minio/baremetal/lifecycle-management/create-lifecycle-management-expiration-rule.html {
+     return 301 https://min.io/minio/docs/linux/administration/object-management/create-lifecycle-management-expiration-rule.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/replication/replication-overview.html {
+     return 301 https://min.io/minio/docs/linux/administration/bucket-replication.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/bucket-replication-overview.html {
+     return 301 https://min.io/minio/docs/linux/administration/bucket-replication.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/replication/enable-server-side-one-way-bucket-replication.html {
+     return 301 https://min.io/minio/docs/linux/administration/bucket-replication/enable-server-side-one-way-bucket-replication.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/replication/enable-server-side-two-way-bucket-replication.html {
+     return 301 https://min.io/minio/docs/linux/administration/bucket-replication/enable-server-side-two-way-bucket-replication.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/replication/enable-server-side-multi-site-bucket-replication.html {
+     return 301 https://min.io/minio/docs/linux/administration/bucket-replication/enable-server-side-one-way-bucket-replication.html?ref=docs-redirect;
```

Comment:
THis should go to
https://min.io/docs/minio/linux/administration/bucket-replication/enable-server-side-multi-site-bucket-replication.html